otelx/config.schema.json

```diff
@@ -56,6 +56,11 @@
                   "description": "The address of jaeger-agent's HTTP sampling server",
                   "format": "uri",
                   "examples": ["http://localhost:5778/sampling"]
+                },
+                "trace_id_ratio": {
```

Comment:
I believe this needs to be listed in line 50 as well.

Reply:
Oops! I forgot to commit this file 🙈